Tweaks to Assessment Documentation (#1124)
fairlearn/fairlearn: docs/user_guide/assessment/intersecting_groups.rst

is that some of the intersections may have very few members (or even be empty).
This will affect the confidence interval associated with the computed metrics;
random noise has a greater effect on smaller groups.

> For this reason, we strongly suggest that :meth:`count` be included as a metric

Author: The `:meth:` bit should make it link to the `count()` metric function in the API docs.

Author: Actually it is now that I think about it.  This isn't a method on a class, but a regular function.

FYI, these are the various commands I'm using:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html?highlight=class#cross-referencing-python-objects

> For this reason, we strongly suggest that :meth:`count` be included as a metric

Reviewer: Ah, my apologies! I thought this was a typo.

> For this reason, we strongly suggest that :func:`count` be included as a metric

Reviewer 2: Nitpick:
```suggestion
To identify such cases, we strongly suggest that :func:`count` be included as a metric
```
